ampd/src/tofnd/mod.rs

```diff
@@ -7,9 +7,10 @@ use crate::url::Url;
 
 pub mod client;
 pub mod error;
+mod grpc;
```

Comment:
```suggestion
pub mod grpc;
```

Reply:
Basically, hide the proto-generated code and initialize the MultisigClient defined in grpc mod in lib.rs